### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Divinity-dev**
Added a pop-up page to display the properties of my projects

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,21 +1,86 @@
-document.getElementById('open-bar').onclick = function() {
```
Comment (Rustamxon7) [inline index.js:1]: Your image            |  Original image
:-------------------------:|:-------------------------:
 |  

You did a great job however your design makes some differences compared to the original design. Kindly fix them step by step.

- [ ]   1. Increase the size of the title. 📐
- [ ]   2. Reduce the size of the close button. ❎
- [ ]   3. Expand the width of the content in relation to the original. 👈
- [ ]   4. Ensure that the See Live and See Source buttons are aligned and sized appropriately. 
- [ ]   5. Utilize any remaining space by filling it inappropriately. 🔭
- [ ]   6. Increase font size for improved legibility. 🔦

File: index.js [hunk 2]
```diff
@@ -1,21 +1,86 @@
...
-
-document.getElementById('toggle').onclick=function(){
+/* eslint-disable prefer-destructuring */
+document.getElementById('open-bar').onclick = () => {
+  document.getElementById('menu-bar').classList.add('show');
+  document.getElementById('open-bar').classList.add('close');
+  document.getElementById('close-bar').classList.remove('close');
+};
+
+document.getElementById('close-bar').onclick = () => {
```
Comment (Rustamxon7) [inline index.js:8]: Your image            |  Original image
:-------------------------:|:-------------------------:
Screenshot 2023-01-03 at 18 29 21 |  

- [ ] 1. Your stack is not in line with the original design parameters; please adjust its dimensions accordingly. 🔧 
- [ ] 2. Please reduce the size of the close button. ❎
- [ ] 3. Please ensure that the font size matches that of the original design. 📐
- [ ] 4. The "See Live" and "See Sources" buttons are not positioned by the initial design, kindly rectify this where necessary. 📝
- [ ] 5. Kindly eliminate any superfluous, wide empty spaces at the bottom of the page. 🪐

File: index.js [hunk 3]
```diff
@@ -1,21 +1,86 @@
...
+  document.getElementById('close-bar').classList.add('close');
+};
+
+document.getElementById('toggle').onclick = () => {
+  document.getElementById('menu-bar').classList.remove('show');
+  document.getElementById('open-bar').classList.remove('close');
+  document.getElementById('close-bar').classList.add('close');
+};
+
+document.getElementById('toggle1').onclick = () => {
```
Comment (Rustamxon7) [inline index.js:20]: - [ ] I cannot open this project's modal menu kindly make sure they are also accessible. Kindly watch the video:

mobile (1)

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hello Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @Divinity-dev, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] the linter files do not run on the PR.
- [ ] that has conflict that need to be resolved. Kindly look into these two issues and address them accordingly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

conflict

**Comment by Divinity-dev** [comment]
I've tried severally to resolve the conflict but it seems the only way I could resolve it is to merge the files which isn't permitted until you give the approval. What do I do?

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 4, 2023 at 11:25 AM, Qoosim ***@***.***> wrote:   


@Qoosim commented on this pull request.

Status: Invalid Request 🔴
 
Hi @Divinity-dev,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request

You have submitted a project, that
   
   -  the linter files do not run on the PR.
   -  that has conflict that need to be resolved. Kindly look into these two issues and address them accordingly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

File: index.html [hunk 4]
```diff
@@ -82,7 +82,7 @@ <h3>
                     <li>Ruby</li>
                 </ul>
                 </div>
-                <button>See Project</button>
+                <button id="portfolio-btn">See Project</button>
```
Comment (MahmoudBakr23) [inline index.html:85]: - [ ] Well done so far on modifying your design as requested by the reviewer. However, please look at the image below and add the missing elements to match the design! There's a missing button please add it.
Screenshot (1)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Divinity-dev 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

n/a

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
What button is missing? I don't see any missing button. Pls show me pictures so I can add the missing button 

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 4, 2023 at 5:41 PM, Mahmoud ***@***.***> wrote:   


@MahmoudBakr23 requested changes on this pull request.

Hi @Divinity-dev 😃 ,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄
 
Required Changes ♻️
 
Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application.

n/a

Cheers and Happy coding! 👏 👨‍💻 💻
 
Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In index.html:
> @@ -82,7 +82,7 @@ <h3>
                     <li>Ruby</li>
                 </ul>
                 </div>
-                <button>See Project</button>
+                <button id="portfolio-btn">See Project</button>
   
   -  Well done so far on modifying your design as requested by the reviewer. However, please look at the image below and add the missing elements to match the design! There's a missing button please add it.

**Comment by MahmoudBakr23** [comment]
@Divinity-dev
The original design picture is shown by the previous reviewer, please take a look at the first review. 😊🙏🏻

**Comment by Divinity-dev** [comment]
The picture you uploaded here isn't my pop up. You see my pop up when you press that button. Press the button and see

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 4, 2023 at 7:14 PM, Mahmoud ***@***.***> wrote:   


@Divinity-dev
The original design picture is shown by the previous reviewer, please take a look at the first review. 😊 🙏🏻

**Review by bimbolabuari — COMMENTED** [review]
Hello @Divinity-dev  👋,

_This is an additional review due to the second opinion request._

- Nice implementation of the modal pop-up  👍 
  - Currently, I'm able to view the modal pop-up for the project cards.


https://user-images.githubusercontent.com/66112760/210636697-a2d5948d-8f1f-4a25-a6c6-f61d7d83d611.mov




## Optional suggestion
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they were mentioned by the first code reviewer and but the previous CR was not consistent with the first review_

- Please work on the pop-up responsiveness, currently, it does not match the required template design  ✊ 

As this was the only reason your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 4, 2023 at 8:47 PM, Abimbola ***@***.***> wrote:   


@bimbolabuari commented on this pull request.

Hello @Divinity-dev 👋 ,

This is an additional review due to the second opinion request.
   
   - Nice implementation of the modal pop-up 👍       
      - Currently, I'm able to view the modal pop-up for the project cards.


https://user-images.githubusercontent.com/66112760/210636697-a2d5948d-8f1f-4a25-a6c6-f61d7d83d611.mov

Optional suggestion

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they were mentioned by the first code reviewer and but the previous CR was not consistent with the first review
   
   - Please work on the pop-up responsiveness, currently, it does not match the required template design ✊

As this was the only reason your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀 !

**Review by Rustamxon7 — APPROVED** [review]
Hi @Divinity-dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
Kindly add the MIT license to your readme :)

Also, you are free to remove this line from your readme:

> NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 4, 2023 at 9:06 PM, Rustam (Rostam) ***@***.***> wrote:   


@Rustamxon7 approved this pull request.

Hi @Divinity-dev,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 


Highlights
   
   - No Linters errors 💯
   - All requirements are complete 💯

Optional suggestions

Kindly add the MIT license to your readme :)

Also, you are free to remove this line from your readme:


NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.


Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**